### PR TITLE
Add iHeartCustom UseType

### PR DESCRIPTION
**Pull request by BWStearns**

### CODE REVIEW

File: pipeline/schema.py [hunk 1]
```diff
@@ -33,7 +33,8 @@
     'ConditionalDownload',
     'NonInteractiveStream',
     'OnDemandStream',
-    'PermanentDownload'
+    'PermanentDownload',
+    'iHeartCustom'
```
Comment (e-agbanyo) [inline pipeline/schema.py]: The rest of this is alphabetical, this change should follow that

Comment (BWStearns) [reply pipeline/schema.py]: I had put it at the end since it's different than the others but I guess alpha works too.

**Comment by BWStearns** [comment]
@e-agbanyo I made the changes

**Comment by e-agbanyo** [comment]
Fixed travis issues. Close duplicate branch.

**Pull request closed without merging**